### DISCUSSION TITLE
Model number changed

### DISCUSSION THREAD

**Discussion by Ak-del-galaxy** [tribalfs/GalaxyMaxHzPub#231] [Q&A]
Hello there i installed galaxymaxhz on my Galaxy note 10 plus via github and i dont know how this changed my model number from **sm-n975/ds** to **PJD110/DS** i want my **sm-n975/ds** back kindly help me community

**Comment by tribalfs** [comment]
GMH doesn't have the capability to change model number. It must be something else.
